.eslintrc.json

```diff
@@ -13,7 +13,8 @@
   "rules": {
     "no-shadow": "off",
     "no-param-reassign": "off",
-    "eol-last": "off"
+    "eol-last": "off",
+    "prefer-destructuring": ["error", {"object": true, "array": false}]
```

Comment:
- [ ] Kindly update your github/workflows to use the html-css-js linter.yml config, so you can check javascript linter too😎 you can find it [here ](https://github.com/microverseinc/linters-config)

Reply:
If you read about this linter issue, you will realise that even if it prefers destructing the object they're instances when you can not use destructing because of the nature of the object. Could you kindly provide more info on what I should do?
[Here](https://stackoverflow.com/questions/54828209/use-array-destructuring-prefer-destructuring-error-on-eslint) is a link to one of the comments in about the same.